src/main/java/com/example/demo/repository/AlterarSenhaDTO.java

```diff
@@ -0,0 +1,34 @@
+package com.example.demo.repository;
+
+public class AlterarSenhaDTO {
+
+    private String email; // E-mail do usuário que quer trocar a senha
+    private String novaSenha; // Nova senha a ser cadastrada
```

Comment:
Tenta usar aquelas anotações de validações que tinha te falado. 
P senha pode usar algo como o pessoal de cadastro vai requerer p preencher a senha (Verifica com eles)
```suggestion
    @Pattern(regexp = "^(?=.*[a-z])(?=.*[0-9])(?=.*[@%$&*?#/|])(?=.*[A-Z])[a-zA-Z0-9@%$&*?#]{8}$",message= "A senha deve conter pelo menos uma letra minúscula e uma maiúscula, um número e um caractere (@%$&*?)")
    @Size(min = 8, message ="A senha deve conter pelo menos 8 dígitos")
    private String novaSenha; // Nova senha a ser cadastrada
```
Isso serve p confirmarSenha tbm.